src/ansible_navigator/actions/_actions.py

```diff
@@ -52,54 +59,95 @@ def _import_all(package: str) -> None:
+
+    :param cls: The class to register
+    :returns: The class after registration
+    """
     package, _, action = cls.__module__.rpartition(".")
     pkg_info = _ACTIONS.setdefault(package, {})
     pkg_info[action] = ActionT(name=action, cls=cls, kegex=re.compile(cls.KEGEX))
     return cls
 
 
 def get(package: str, action: str) -> Callable:
-    """Get a given action"""
+    """Import and return a given action.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :returns: The action's registered class
+    """
     _import(package, action)
     return _ACTIONS[package][action].cls
 
 
 def get_factory(package: str) -> Callable:
-    """Create a get() function for one package"""
+    """Create a ``get`` function for one package.
+
+    :param package: The name of the package
+    :returns: The action's registered class
+    """
     return functools.partial(get, package)
 
 
 def kegex(package: str, action: str) -> Tuple:
-    """Return a tuple of name, kegex for a action"""
+    """Return a tuple of name, class, kegex for a action.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :returns: The name, class and kegex for an action
+    """
     _import(package, action)
     return _ACTIONS[package][action]
 
 
 def kegexes(package: str) -> Generator:
-    """Return a tuple of tuples, name, kegex"""
+    """Return a tuple of tuples, name, kegex for all actions.
+
+    :param package: The name of the package
+    :returns: A generator for all kegexs
+    """
     _import_all(package)
     return (kegex(package, name) for name in names(package))
 
 
 def kegexes_factory(package: str) -> Callable:
-    """Create a kegexs() function for all packages"""
+    """Create a ``kegexs`` function for all packages.
+
+    :param package: The name of the package
+    :returns: A ``kegexes`` method for the package
+    """
     return functools.partial(kegexes, package)
 
 
 def names(package: str) -> List:
-    """List all actions in one package"""
+    """List all actions in one package.
+
+    :param package: The name of the package
+    :returns: All packages
+    """
     _import_all(package)
     return sorted(_ACTIONS[package])
 
 
 def names_factory(package: str) -> Callable:
-    """Create a names() function for one package"""
+    """Create a ``names`` function for one package.
+
+    :param package: The name of the package
+    :returns: a ``names`` method for the package
+    """
     return functools.partial(names, package)
 
 
 def run_interactive(package: str, action: str, *args: Any, **_kwargs: Any) -> Any:
-    """Call the given action's run"""
+    r"""Call the given actions's run method.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :param \*args: The arguments passeed to the action's run method
+    :param \**_kwargs: The keyword arguments passed to the action's run method
```

Comment:
I don't think it's customary to use asterisks like that. Have you seen it anywhere else?
```suggestion
    :param _kwargs: The keyword arguments passed to the action's run method
```

Reply:
I did some googling and while Sphinx's docs don't describe this separately at https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists, its own source code has examples like https://github.com/sphinx-doc/sphinx/blob/f38bd8e9529d50e5cceffe3ca55be4b758529ff7/sphinx/application.py#L558 and it's rendered like this: https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_node (without asterisks in the parameter description area but the signature itself has asterisks).

Reply:
Same with `*args`: https://github.com/sphinx-doc/sphinx/blob/f38bd8e9529d50e5cceffe3ca55be4b758529ff7/sphinx/application.py#L440.

Reply:
all cleaned up, not sure how I got in that rat hole.

Reply:
You probably googled and found some of the StackOverflow answers + docutils escaping guidelines. I've seen those too but then, I tried to look into what sphinx does and found that it doesn't bother to add asterisks — they are not a part of var names anyway.

Reply:
Besides, it's the same as with type annotations — they annotate what gets into those variables in runtime.